microsoft-edge/progressive-web-apps/samples/temperature-converter.md

```diff
@@ -0,0 +1,599 @@
+
+![The sample Progressive Web App (PWA), installed and running in its own window](./index-images/sample-pwa-installed.png)
+
+To learn more about installing Progressive Web Apps (PWAs), see [Use PWAs in Microsoft Edge](../ux.md).
+
+Continue with the next section.
+
+
+<!-- ====================================================================== -->
+## Run the sample PWA as a local app
+
+Run the sample Progressive Web App (PWA) as a local app on the device.
+
+1. Press the **Windows** key, start typing **Temperature converter app**, and then click **Open**.
+
+   The installed **Temperature converter app** opens:
+
+   ![The sample PWA, installed and running in its own window](./temperature-converter-images/sample-pwa-installed.png)
+
+   The service worker (`sw.js`) is running.  The localhost server is automatically running<!-- todo: is npx server auto-started? -->.
+
+   Because you installed the app from `localhost`, when you start the app, the right side of the title bar displays **localhost:8080** for a few seconds:
+
+   ![The installed sample Progressive Web App (PWA) if installed from localhost](./temperature-converter-images/installed-from-localhost.png)
+
+   If you had installed the app from the `github.io` server, when you start the app, the right side of the title bar would instead display **microsoftedge.github.io** for a few seconds:
+
+   ![The installed sample Progressive Web App (PWA) if installed from github.io server](./temperature-converter-images/installed-from-github-io.png)
+
+1. In the **temperature** text box, type **22**.
+
+   As you type, the displayed temperature is updated to **71.6 F**.
+
+1. Click the **from** and **to** dropdown lists.
+
+   Available units are:
+   * **Celsius**
+   * **Fahrenheit**
+   * **Kelvin**
+
+Continue with the next section.
+
+
+<!-- ====================================================================== -->
+## Use DevTools to examine the service worker handling offline caching
+
+The local app window for a Progressive Web App (PWA) is a browser-based window, so Microsoft Edge DevTools can be used to work with the PWA.
+
+Confirm that the service worker (`sw.js`) is running after you open the PWA as a local app, as follows:
+
+1. Right-click the app, and then select **Inspect**.
+
+   DevTools opens, in a separate window; undocked.  Other docking options are dimmed and not available.
+
+1. In DevTools, select the **Application** (![Application icon](./temperature-converter-images/application-tool-icon.png)) tool.
+
+1. In the tree on the left, select **Application** > **Service workers**.
+
+   Information about the service worker is displayed.  The service worker's **Source** is `sw.js`, with **Status** of **activated and is running**:
+
+    ![The Application tool (in DevTools), showing the "Service workers" panel, with the `sw.js` worker running](./temperature-converter-images/devtools-service-workers.png)
```

Comment:
* blurry - make window narrow then redo
* also show undocked DevTools window

https://review.learn.microsoft.com/en-us/microsoft-edge/progressive-web-apps/samples/temperature-converter?branch=pr-en-us-3476#use-devtools-to-examine-the-service-worker-handling-offline-caching